src/runtime_src/core/pcie/windows/device_windows.cpp

```diff
@@ -697,6 +709,12 @@ struct devinfo
         //sample strings: xilinx_u250_GOLDEN, xilinx_u250_gen3x16_base, xilinx_u250_xdma_201830_3
         return (shell.find("GOLDEN") != std::string::npos) ? false : true;
       }
+    case key_type::mac_addr_first:
+      return std::string(info.MacAddrFirst);
+    case key_type::mac_contiguous_num:
+      return info.MacContiguousNum;
+    case key_type::mac_addr_list:
+      return std::vector<std::string>{ std::string(info.MacAddr0), std::string(info.MacAddr1), std::string(info.MacAddr2), std::string(info.MacAddr3) };
```

Comment:
```suggestion
      return { std::string(info.MacAddr0), std::string(info.MacAddr1), std::string(info.MacAddr2), std::string(info.MacAddr3) };
```

Reply:
Observing compile time error when tried to add your suggestion in the code

```
 C:\Users\Administrator\Desktop\1july_xrt\XRT\src\runtime_src\core\pcie\windows\device_windows.cpp(757): error C2440:
'return': cannot convert from 'initializer list' to 'boost::any' [C:\Users\Administrator\Desktop\1july_xrt\XRT\build\WR
elease\runtime_src\core\pcie\windows\xrt_core.vcxproj]
```

Reply:
Sorry, my bad: I missed that the function was returning a `boost::any`.